### PR TITLE
Add link to AutoClearedValue

### LINKED ISSUE #330: Add AutoCleardValue (Google) to README
## Kind (Required)

- Improvement

 ## Overview (Required)

- AutoCleardValue is used for clear binding in Fragment

 ## Links	

-

### PR DESCRIPTION
**Pull request by andooown**
## Issue
- close #330

## Overview (Required)
- Add link to AutoClearedValue to README
  - AutoClearedValue from Android Architecture Components samples

## Links
- [architecture\-components\-samples/AutoClearedValue\.kt at master · android/architecture\-components\-samples](https://github.com/android/architecture-components-samples/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/util/AutoClearedValue.kt)

### CODE REVIEW

**Comment by satsukies** [comment]
👀

**Comment by takahirom** [comment]
👀

File: README.md [hunk 1]
```diff
@@ -228,6 +228,8 @@ This project uses some modern Android libraries and source codes.
 * [Injected ViewModel Provider](https://github.com/evant/injectedvmprovider) (evant)
 * [Google I/O 2018](https://github.com/google/iosched) (Google)
 * [TimetableLayout](https://github.com/MoyuruAizawa/TimetableLayout) (MoyuruAizawa)
+* [Android Architecture Components samples](https://github.com/android/architecture-components-samples) (Google)
+  * [AutoClearedValue](https://github.com/android/architecture-components-samples/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/util/AutoClearedValue.kt)
```
Comment (takahirom) [inline README.md]: Can you use link specific commit? This link uses master. When the file removed we can not access the page.
You can access URL when pressing `y` on the page.
https://github.com/android/architecture-components-samples/blob/master/GithubBrowserSample/app/src/main/java/com/android/example/github/util/AutoClearedValue.kt

Comment (andooown) [reply README.md]: I fixed it to specified commit used in [this app](https://github.com/DroidKaigi/conference-app-2020/blob/5dbba2f359a713a8e5181625d6ffbdc4598171d9/corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/util/AutoClearedValue.kt).

And the tips of shortcut (`y`) is help for me. Thanks for review!

**Pull request merged**